### PR TITLE
Additional minion options

### PR DESCRIPTION
**Pull request by lomeroe**
add ability to specify additional options to the minion config file during converge

### CODE REVIEW

**Comment by amontalban** [comment]
Would be great to have this merged!

File: provisioner_options.md [hunk 1]
```diff
@@ -290,3 +291,13 @@ For example, the following suite will define grains on the guest:
 
 ### [vendor_path](id:vendor_path)
 
+### [additional_minion_config](id:additional_minion_config)
+The additional_)minion_config option allows you to specify additional options to set in the minion's config file.
```
Comment (amontalban) [inline provisioner_options.md:295]: The additional_minion_config option allows you to specify additional options to set in the minion's config file.

**Comment by amontalban** [comment]
@lomeroe can you update the PR?

@simonmcc @jbussdieker any chance we can merge this? I think it's very useful so we can pass minion config options in the kitchen.yml file.

Thanks!

**Comment by gtmanfred** [comment]
@lomeroe does #93 fit your need here?

Thanks,
Daniel

**Comment by lomeroe** [comment]
at first glance, it appears I could adapt to it

**Comment by gtmanfred** [comment]
Awesome, I am going to go ahead and close this one then, and if you come back and want to rework this, i would be more than happy to get it merged in.

Thanks,
Daniel

**Pull request closed without merging**